### ISSUE TITLE
Setting OptimizingObjective in ompl based planning

### ISSUE THREAD

**Issue by simonschmeisser** [moveit/moveit_planners#58]
It would be nice to be able to set the OptimizingObjective when using ompl-based planing. Currently my robot arm does some pretty wild movements at times and optimizing for mechanical work might give better results?

http://ompl.kavrakilab.org/optimalPlanning.html

http://ompl.kavrakilab.org/optimalPlanningTutorial.html

**Comment by davetcoleman** [comment]
Yes, that would be a very appreciated feature. I hacked a version of this years ago, but its long gone and I've never gotten a chance to set it up correctly.

**Comment by rbbg** [comment]
I have a pretty recent fork that allows this over at https://github.com/rbbg/moveit_planners. Not sure if it is the most appropriate way to do it, but perhaps it's a good idea to submit a pull request to at least get a discussion going. It's not a huge change and I have seen people asking about it at least a number of  times.

**Comment by davetcoleman** [comment]
Yes, a pull request is a good way to get the discussion started
